### PR TITLE
DOCS-9161-db-index-studio-template-kt

### PR DESCRIPTION
**Pull request by kevintroller**
Database Connector reorganization:
Put Index.adoc and database-studio-configuration.adoc into new Connectors template.

### CODE REVIEW

File: db/1.8/modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .xref:index.adoc[Database Connector]
 * xref:index.adoc[About Database Connector]
-* xref:database-connector-design-center.adoc[Database Design Center Configuration]
 * xref:database-connector-studio.adoc[Database Studio Configuration]
-* xref:database-connector-xml-maven.adoc[Database XML and Maven Support]
-* xref:database-connector-examples.adoc[Database Examples]
 * xref:database-documentation.adoc[Database Connector Reference]
+* xref:database-connector-examples.adoc[Database Examples]
```
Comment (valkyrie69) [inline db/1.8/modules/ROOT/nav.adoc]: I usually put Connector in the title  Database Connector Examples, Database Connector XML and Maven Support, etc
And then for the Design Center topic, Use Flow Designer to Configure Database Connector or something like that.

File: db/1.8/modules/ROOT/pages/database-connector-studio.adoc [hunk 2]
```diff
@@ -1,32 +1,107 @@
...
+== Configure a Global Element for the Connector
+
+When you configure a connector, it’s best to configure a global element that all instances of that connector in the app can use. Database Connector can connect to any database for which a JDBC driver is available. To make it easier to connect to the most popular databases (such as MySQL, Oracle, and Microsoft SQL Server), the connector includes a series of different connection types.
+
+To configure the global element for Database Connector, follow these steps:
+
+. Select the operation in the Studio canvas.
+. In the configuration screen for the operation, click the plus sign (*+*) next to the *Connector configuration* field to access the global element configuration fields.
+. On the *Database Config* window, for *Connection*, select any of the connection types to provide to this configuration.
+. On the *General* tab, specify the database connection information for the connector.
```
Comment (valkyrie69) [inline db/1.8/modules/ROOT/pages/database-connector-studio.adoc:109]: For the connection configuration, we usually list the available connection types--in this case:
Derby
Generic
Microsoft SQL
MySQL
Oracle
etc

File: db/1.8/modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -1,78 +1,66 @@
...
 
-You can perform predefined queries, dynamically constructed queries, and template queries that are self-sufficient and customizable. You can perform multiple SQL requests in a single bulk update and make Data Definition Language (DDL) requests that alter the data structure rather than the data itself.
+* Derby
+* Microsoft SQL Server
+* MySQL
+* Oracle
 
-Release Notes: xref:release-notes::connector/connector-db.adoc[Database Connector Release Notes] +
-Exchange: https://www.mulesoft.com/exchange/org.mule.connectors/mule-db-connector/[Database Connector]
+You can set up other JDBC databases using a generic JDBC configuration. Additionally, you can reference a JDBC `DataSource` object or a `XADataSource` object, which is required for XA transactions by creating the object in Anypoint Studio using Spring.
```
Comment (valkyrie69) [inline db/1.8/modules/ROOT/pages/index.adoc]:
```suggestion
You can set up other JDBC databases using a generic JDBC configuration. Additionally, you can reference a JDBC `DataSource` object or an `XADataSource` object, which is required for XA transactions, by creating the object in Anypoint Studio using Spring.
```

File: db/1.8/modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -1,78 +1,66 @@
-= Database Connector - Get Started - Mule 4
+= Database Connector 1.8 - Mule 4
 :page-aliases: connectors::db/db-configure-connection.adoc, connectors::db/db-connector-index.adoc
 
-
 Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
```
Comment (valkyrie69) [inline db/1.8/modules/ROOT/pages/index.adoc:5]: Are there Prerequisites?

File: db/1.8/modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -1,78 +1,66 @@
...
+* Derby
+* Microsoft SQL Server
+* MySQL
+* Oracle
 
-Release Notes: xref:release-notes::connector/connector-db.adoc[Database Connector Release Notes] +
-Exchange: https://www.mulesoft.com/exchange/org.mule.connectors/mule-db-connector/[Database Connector]
+You can set up other JDBC databases using a generic JDBC configuration. Additionally, you can reference a JDBC `DataSource` object or a `XADataSource` object, which is required for XA transactions by creating the object in Anypoint Studio using Spring.
 
+== Common Use Cases for the Connector
```
Comment (valkyrie69) [inline db/1.8/modules/ROOT/pages/index.adoc:29]: Add a lead sentence for the bullet list.

File: db/1.8/modules/ROOT/pages/index.adoc [hunk 6]
```diff
@@ -1,78 +1,66 @@
...
+* Perform multiple SQL requests in a single bulk update.
+* Make Data Definition Language (DDL) requests that alter the data structure rather than the data itself.
+* Execute stored procedures and SQL scripts.
 
-Anypoint connectors are Mule runtime engine extensions that enable you to connect
-to APIs and resources on external systems, such as Salesforce, Database, ServiceNow,
-and Twitter.
+For examples, see xref:database-connector-examples.adoc[Examples].
 
 == Prerequisites
```
Comment (valkyrie69) [inline db/1.8/modules/ROOT/pages/index.adoc]: Oh, here are the prereqs.  Move this up so it's before the common uses cases.

File: db/1.8/modules/ROOT/pages/index.adoc [hunk 7]
```diff
@@ -1,78 +1,66 @@
...
 
+== Next Step
 
-== Next
-
-After you have met the prerequisites and experimented with templates
-and examples, you are ready to create an app with
-xref:database-connector-design-center.adoc[Design Center] or
-xref:database-connector-studio.adoc[Anypoint Studio].
+After you complete the prerequisites and try the templates and examples, you are ready to create your own app and configure the connector using xref:database-connector-studio.adoc.adoc[Anypoint Studio].
```
Comment (valkyrie69) [inline db/1.8/modules/ROOT/pages/index.adoc]: ugh...this is not really the next step. We need to update this.  :-)

**Review by valkyrie69 — DISMISSED** [review]

**Review by emariotti3 — DISMISSED** [review]

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**